### DISCUSSION TITLE
Select site for entry link

### DISCUSSION THREAD

**Discussion by lukas-jansen** [craftcms/ckeditor#89] [Ideas] [answered]
In redactor you had the option to select to which site you wanted to link when selecting an entry (currentsite, site A, site B,...), In CkEditor this seems not possible and always create a link to to entry that specific sets the site to the current site. When you than have a multi site setup and you don't translate the link it will link to the wrong site.

**Comment by brandonkelly** [comment, marked as answer]
Added a site selection dropdown for the next release! (21ee41a854a7cb40c9eb29956c90711c16c15e58)

<img src="https://user-images.githubusercontent.com/47792/235741060-f4cd56ca-4e4f-49d3-95ed-ed95b9d0f1f2.png" width="352" height="220" alt="A link edit form within a CKEditor field, showing that the form contains a site selection dropdown">

Like Redactor, the dropdown contains a “Link to the current site” option, which removes the `@<SiteID>` bit from the reference tag.

**Reply by brandonkelly** [reply]
3.3.0 is out with that change 🎉

**Comment by yeboJoseph** [comment]
@brandonkelly, would you be able to allow us to set the default as "Link to current site"
